vcbuild.bat

```diff
@@ -386,6 +393,27 @@ ssh -F %SSHCONFIG% %STAGINGSERVER% "touch nodejs/%DISTTYPEDIR%/v%FULLVERSION%/no
 if errorlevel 1 goto exit
 
 
+:build-doc
+@rem Build documentation if requested
+if not defined doc goto run
+if not exist %node_exe% (
+  echo Failed to find node.exe
+  goto run
+)
+mkdir %config%\doc
+robocopy /e doc\api %config%\doc\api
+robocopy /e doc\api_assets %config%\doc\api\assets
+
+if exist "tools\doc\node_modules\js-yaml\package.json" goto doc-skip-js-yaml
+cd tools\doc
+%npm_exe% install
+cd ..\..
+:doc-skip-js-yaml
+for %%F in (%config%\doc\api\*.md) do (
+  %node_exe% tools\doc\generate.js --format=json %%F > %%~dF%%~pF%%~nF.json
+  %node_exe% tools\doc\generate.js --node-version=v%FULLVERSION% --format=html --template=doc\template.html --analytics=%DOCS_ANALYTICS% %%F > %%~dF%%~pF%%~nF.html
```

Comment:
It seems the https://github.com/nodejs/node/pull/20307 is approved so `--template=doc\template.html` can be deleted after that PR is landed to sync with the `Makefile` change.